src/App.js

```diff
@@ -1,23 +1,46 @@
-import logo from './logo.svg';
+import { useEffect, useState } from 'react';
+import { getPokemon } from './services/fetch-utils.js';
 import './App.css';
 
 function App() {
+  const [pokemon, setPokemon] = useState([]);
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
I'd recommend defaulting loading to true, even if your `useEffect` takes care of that first